app/services/foreman_rh_cloud/cert_auth.rb

```diff
@@ -12,10 +12,18 @@ def cert_auth_available?(organization)
     def execute_cloud_request(params)
       organization = params.delete(:organization)
       certs = ForemanRhCloud.with_iop_smart_proxy? ? foreman_certificate : candlepin_id_cert(organization)
-      final_params = {
+      default_params = {
         ssl_client_cert: OpenSSL::X509::Certificate.new(certs[:cert]),
         ssl_client_key: OpenSSL::PKey.read(certs[:key]),
-      }.deep_merge(params)
+      }
+
+      if ForemanRhCloud.with_iop_smart_proxy? && organization&.label
+        default_params[:headers] ||= {
```

Comment:
This should be sufficient
```suggestion
        default_params[:headers] = {
```